psa-crypto-sys/build.rs

```diff
@@ -388,6 +397,7 @@ mod operations {
             let (from, _) = &lib;
             args.push(from.as_os_str());
         }
+        println!("BILL: args: {:?}", args);
```

Comment:
Can this be dropped?

Reply:
Yeah whoops, I thought I dropped all of those :-p